pkgs/build-support/node/build-npm-package/default.nix

```diff
@@ -1,4 +1,11 @@
-{ lib, stdenv, fetchNpmDeps, buildPackages, nodejs } @ topLevelArgs:
+{ lib
+, stdenv
+, fetchNpmDeps
+, buildPackages
+, nodejs
+, cctools
+, python3
```

Comment:
I wonder if we should use `nodejs.python` instead of accepting python3 as a callPackage argument to ensure it's always using the same python as the node-gyp from nodejs?

Reply:
Thanks, I changed `python3` to `nodejs.python`.